### PR TITLE
{bio,tools}[fosscuda/2019b] Bonito v0.1.0, apex v20200325 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2407.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0a4c114cdf4a4c03f376d3d9b684935e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 3 (2 easyconfigs in this PR)
node3117.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/59b1ab3f7878996c48689c176e700deb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3302.joltik.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, Python 3.6.8
See https://gist.github.com/34cfa60828709ee72daa7893b33aa06e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2404.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6a9f58acf7a62e6bec6ac780eece8299 for a full test report.

File: easybuild/easyconfigs/b/Bonito/Bonito-0.1.0-fosscuda-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+description = "Convolution Basecaller for Oxford Nanopore Reads"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+local_pymajver = '3'
+local_pyminver = '7'
+local_pymajminver = local_pymajver + local_pyminver
+
+dependencies = [
+    ('Python', '%s.%s.4' % (local_pymajver, local_pyminver)),
```
Comment (lexming) [inline easybuild/easyconfigs/b/Bonito/Bonito-0.1.0-fosscuda-2019b-Python-3.7.4.eb:17]: These local variables seem unnecessary as they are not used anywhere else in the easyconfig
```suggestion
dependencies = [
    ('Python', '3.7.4'),
```

File: easybuild/easyconfigs/b/Bonito/Bonito-0.1.0-fosscuda-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    }),
+    ('fast-ctc-decode', '0.2.3', {
+        'source_tmpl': 'fast_ctc_decode-0.2.3-cp37-cp37m-manylinux1_x86_64.whl',
+        'unpack_sources': False,
+        'checksums': ['20aa7a30ee6023ed0c20f583c7af2c27f49a812700e9cf9f21f540650f6e62cb'],
+    }),
+    ('ont-bonito', version, {
+        'modulename': 'bonito',
+        'source_tmpl': 'ont-bonito-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/o/ont-bonito'],
```
Comment (lexming) [inline easybuild/easyconfigs/b/Bonito/Bonito-0.1.0-fosscuda-2019b-Python-3.7.4.eb:90]: These `source_tmp` and `source_urls` look like the default values
```suggestion
```

File: easybuild/easyconfigs/b/Bonito/Bonito-0.1.0-fosscuda-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/b/Bonito/Bonito-0.1.0-fosscuda-2019b-Python-3.7.4.eb]: fixed in 84156b5f7c

File: easybuild/easyconfigs/b/Bonito/Bonito-0.1.0-fosscuda-2019b-Python-3.7.4.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/b/Bonito/Bonito-0.1.0-fosscuda-2019b-Python-3.7.4.eb]: fixed in 84156b5f7c

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/96a3d8e1da1b0c9a8ff05021b80909e3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3126.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/02ea71a3561a446861f770ec5d9e4071 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in this PR)
node150.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/0e2d54a2d947a2b75894dfec56d656d1 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**